package.json

```diff
@@ -105,10 +103,7 @@
     "@types/webpack": "^4.41.32",
     "@typescript-eslint/eslint-plugin": "^5.10.0",
     "@typescript-eslint/parser": "^5.10.0",
-    "@vue/cli-plugin-babel": "^4.5.15",
-    "@vue/cli-plugin-pwa": "^4.5.15",
     "@vue/cli-plugin-typescript": "^4.5.15",
```

Comment:
I guess we are not using this as well?

Reply:
`@vue/cli-plugin-typescript` removing this breaks the build, so I think its being used internally for some setup